WcaOnRails/app/assets/javascripts/registrations.js

```diff
@@ -58,3 +58,17 @@ function compareHtmlContent(a, b) {
   var second = $('<p>' + b + '</p>').text().trim();
   return first.localeCompare(second);
 }
+
+onPage('registrations', function() {
+    $('#registration_comments').parent('div').append('<p id="comment-length-counter"> 0/280 </p>');
```

Comment:
I'm not in love with `280` being hardcoded here, what about moving `maxlength` above this line and using it here directly?
```js
var textarea = document.getElementById("registration_comments");
var maxlength = textarea.getAttribute("maxlength");
// Create div
```